butler-bot/bot.js

```diff
@@ -63,4 +68,46 @@ client.on('message', message => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+client.setInterval((servers) => {
+  const itMuslimServer = servers.get(IT_MUSLIM_SERVER_ID)
+
+  const incompleteMembers = itMuslimServer.members.filter((member) => {
+    return !member.roles.has(FULL_MEMBER_ROLE_ID)
+  })
+  /* console.log(incompleteMembers) */
+
+  const incompleteMembersName = incompleteMembers.map((item) => {
+    return item.user.username
+  })
+  /* console.log(incompleteMembersName) */
+
+  const membersJoinTime = incompleteMembers.map((member) => {
+    return member.joinedTimestamp
+  })
+  /* console.log(membersJoinTime) */
+
+  /* const greetingsChannel = client.channels.get(GREETINGS_CHANNEL_ID) */
+
+  membersJoinTime.forEach((item) => {
+    let timeNow = new Date()
+    let timeDifference = (timeNow.getTime() - item) / 1000 / 60 / 60 / 24
+
+    for (let i = 0; i < incompleteMembersName.length; i++) {
```

Comment:
Something strange happens below. Just [`kick`](https://discord.js.org/#/docs/main/stable/class/GuildMember?scrollTo=kick) these members. If you will try to do it in TestBot, it will return an error, because it doesn't have permissions to do it.